mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -207,8 +211,7 @@ def test_io_set_raw(fnames, tmpdir):
     with pytest.warns(None) as w:
         read_raw_eeglab(input_fname=one_chan_fname, preload=True)
     # no warning for 'no events found'
-    w.pop(DeprecationWarning)
-    assert len(w) == 0
+    assert len(w) == 0  # XXXX
```

Comment:
we do warn that we have no events. Why shouldn't we? I could not figure out why the old behavior was not warning.